src/Admin/Views/Shared/_OrganizationFormScripts.cshtml

```diff
@@ -15,10 +16,15 @@
                 return;
             }
             if (gateway.value === '@((byte)GatewayType.Stripe)') {
-                window.open('https://dashboard.stripe.com/customers/' + customerId.value, '_blank');
+                const url = '@(HostingEnvironment.IsDevelopment()
```

Comment:
⛏️ All looks good to me! Only thing I can see would be the potential to cut down on the repeated ternary operators with extension methods off the `IHostingEnvironment` like `IHostingEnvironment.GetStripeUrl` and `IHostingEnvironment.GetBraintreeMerchantUrl`.

Reply:
Thanks for the suggestion! I just pushed a change to add this in, so take a look and let me know what you think